kitchen/bin/build-uberjar.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# Usage: build-uberjar.sh
+#
+# Builds the kitchen uberjar.
+
+set -ev
+
+KITCHEN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )"
+export PATH=${KITCHEN_DIR}/..:$PATH
```

Comment:
Can we do this line in `run-integration-tests*.sh` instead of here, since this is inappropriate for local runs?